src/main.rs

```diff
@@ -39,20 +42,21 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 .exec()
                 .await
                 .expect("Cannot fetch gateway information");
+            let concurrency = gateway
+                .model()
+                .await?
+                .session_start_limit
+                .max_concurrency
+                .try_into()
+                .unwrap();
```

Comment:
I don't think it's relevant to remove this unwrap in this PR, but we should probably do it eventually.